src/Instrumentation/Symfony/Messenger/EventSubscriber/EndSpanEventSubscriber.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
There is no need for strict types.

Reply:
you don't want it to be sure the bundle is compatible even with apps running without strict_types=1 ?

Reply:
I do not want to infect any application using this bundle with strict types, I prefer to rely on PHPStan to make things right rather than potentially breaking application side code.

You check this issue from Symfony to get severals pov on the subject: https://github.com/symfony/symfony/issues/28423

Reply:
removed everywhere in the directory `src`